docker-compose.yml

```diff
@@ -24,30 +24,30 @@ services:
       # Allow Redis access from outside the container
       - "63791:6379"
 
-  twoops_tracker:
-    image: codeforafrica/twoops-tracker:latest
+  twoopstracker_app:
+    image: codeforafrica/twoopstracker:latest
     depends_on:
       - db
       - redis
-    environment:
-      - TWOOPSTRACKER_DATABASE_URL=postgres://${PGUSER:-twoops_tracker}:${PGPASSWORD:-twoops_tracker}@db:5432/${PGDATABASE:-twoops_tracker}
-      - TWOOPSTRACKER_CELERY_BROKER_URL=${TWOOPSTRACKER_CELERY_BROKER_URL:-redis://redis:6379/0}
     env_file:
-      - ./twoops_tracker/.env
+      - ./twoops_tracker/.env.template
```

Comment:
The code looks good but I'm not sure about this section. When performing the docker compose up, will the users have to specify the `env_file` manually? Given that we mention the `cp  twoops_tracker/.env.template twoops_tracker/.env` in the readme, shouldn't we just use `twoops_tracker/.env` as the `env_file`?

Reply:
Not a bad idea & I think that's what was originally there. 2 reasons for this switch @esirK :

1. `twoops_tracker/.env` doesn't exist in the repo. This means if we'll be hooking up to a non-existing file which I think isn't very good/robust i.e. broken by design.
2. `docker compose` offers more than [one](https://docs.docker.com/compose/environment-variables/) [way](https://docs.docker.com/compose/envvars-precedence/) to set environment variables so while we suggest (recommend?) using `.env`, it should be possible to use any of the other methods as well without breaking anything.

Reply:
Makes sense.